Packages/Tracking/Core/Runtime/Plugins/Android/Other/arm64-v8a/libLeapC.so

```diff
@@ -0,0 +1,3 @@
+version https://git-lfs.github.com/spec/v1
+oid sha256:2d6d339acfe4991b90b9a3a68774759cc96649481529839212d0ed68c4557b97
+size 5138512
```

Comment:
Can we check tha this is correct, this is a Git LFS header, but it's wierd that it's shown, it's almost like it's been added as a Git LFS file, but not committed as one...

Reply:
On the packaging of HandTrackerServer.framework, I can remove it. I'm keen otherwise to get this merged for Luke.